src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioDiagnosticListSuppressionStateService.cs

```diff
@@ -159,27 +159,9 @@ private static bool EntrySupportsSuppressionState(ITableEntryHandle entryHandle,
         isNoLocationDiagnosticEntry = !entryHandle.TryGetValue(StandardTableColumnDefinitions.DocumentName, out string filePath) ||
             string.IsNullOrEmpty(filePath);
 
-        var roslynSnapshot = GetEntriesSnapshot(entryHandle, out var index);
```

Comment:
see comment on GetEntriesSnapshot.  roslynSnapshot will always be null.  so the first if is taken unilaterally.